lnwallet/channel.go

```diff
@@ -385,6 +385,28 @@ type PaymentDescriptor struct {
 	BlindingPoint lnwire.BlindingPointRecord
 }
 
+// AddHeight returns a pointer to the height at which the HTLC was added to the
+// commitment chain. The height is returned based on the chain the HTLC is
+// being added to (local or remote chain).
+func AddHeight(htlc *PaymentDescriptor, remoteChain bool) *uint64 {
+	if remoteChain {
+		return &htlc.addCommitHeightRemote
+	}
+
+	return &htlc.addCommitHeightLocal
+}
+
+// RemoveHeight returns a pointer to the height at which the HTLC was removed
+// from the commitment chain. The height is returned based on the chain the HTLC
+// is being removed from (local or remote chain).
+func RemoveHeight(htlc *PaymentDescriptor, remoteChain bool) *uint64 {
```

Comment:
Why do these functions return pointer types?

Reply:
It seems like we modify the underlying value in some places. Updated the method's Godoc to mention this.